### PR TITLE
OPENSHIFTP-109/110: Fix for AXFR vulnerability and Disabling Track/Trace for httpd

### PR DESCRIPTION
**Pull request by Chandan-Abhyankar**
Fix for AXFR vulnerability and Disabling Track/Trace for httpd server

### CODE REVIEW

File: templates/named.conf.j2 [hunk 1]
```diff
@@ -40,6 +40,9 @@ options {
 	pid-file "/run/named/named.pid";
 	session-keyfile "/run/named/session.key";
 
+	request-ixfr no;
+	allow-transfer { 1.1.1.1;};
```
Comment (prb112) [inline templates/named.conf.j2:44]:
```suggestion
	allow-transfer { 1.1.1.1;};
	allow-recursion {"none";};
```

Comment (prb112) [reply templates/named.conf.j2:44]: Hey @Chandan-Abhyankar can you add this too? I found this one this weekend. Thanks, paul

**Comment by prb112** [comment]
/assign @Prajyot-Parab 

Hey Prajyot, could you review? If you want, we can put these behind a flag, Thanks, Paul

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Comment by prb112** [comment]
Thank you

**Pull request merged**